packages/runtime/datastore-definitions/api-report/datastore-definitions.api.md

```diff
@@ -89,7 +89,7 @@ export interface IDeltaHandler {
 // @public
 export type IDeltaManagerErased = ErasedType<"@fluidframework/container-definitions.IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>">;
 
-// @public @sealed
+// @alpha @sealed
```

Comment:
Several typer moved from public to alpha in here, including IFluidDataStoreRuntime